keyboards/clueboard/66/config.h

```diff
@@ -15,10 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CLUEBOARD_66_CONFIG_H
-#define CLUEBOARD_66_CONFIG_H
-
-#include "config_common.h"
```

Comment:
I think you still need this, actually.

```suggestion
#include "config_common.h"	
```